src/libse/LibSE.csproj

```diff
@@ -37,12 +37,7 @@
 			<LastGenOutput>Resources.Designer.cs</LastGenOutput>
 		</EmbeddedResource>
 	</ItemGroup>
-
-	<Target Name="InvokeBuildHelpers" BeforeTargets="BeforeCompile;CoreCompile">
```

Comment:
Moved to SubtitleEdit.csproj, the batch script is used to 
generate stuff for UI not libse